src/main/kotlin/uk/gov/justice/digital/hmpps/config/ClientTrackingConfiguration.kt

```diff
@@ -13,8 +15,6 @@ import org.springframework.web.servlet.config.annotation.InterceptorRegistry
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
 import java.text.ParseException
 import java.util.Optional
-import javax.servlet.http.HttpServletRequest
-import javax.servlet.http.HttpServletResponse
 
 @Configuration
 @ConditionalOnExpression("T(org.apache.commons.lang3.StringUtils).isNotBlank('\${applicationinsights.connection.string:}')")
```

Comment:
worth having a look at our new version of this configuration that doesn't use app insights classes - https://github.com/ministryofjustice/hmpps-restricted-patients-api/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/config/ClientTrackingConfiguration.kt#L36

Reply:
same for https://github.com/ministryofjustice/hmpps-restricted-patients-api/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/config/ApplicationInsightsConfiguration.kt

Reply:
you don't need `ConditionalOnExpression` as the default implementation is a no-op

Reply:
sorted